lavorare-insieme/dove-quando-e-come.md

```diff
@@ -0,0 +1,98 @@
+incentrata sul lavoro da remoto. Del resto lavoriamo da remoto per tutti i nostri clienti!
+
+Detto questo, il lavoro da remoto porta con sé delle sfide non indifferenti che è importante 
+affrontare preventivamente, per evitare di influire negativamente sulla produttività propria e di 
+tutto il team. In questa sezione spiegheremo alcune di queste sfide e come si possono vincere.
+
+### Principi fondamentali
+
+Innanzitutto è necessaro tenere sempre presenti alcuni principi fondamentali quando si lavora da
+remoto:
+
+- Il lavoro da remoto non è semplicemente un benefit, ma una vera e propria filosofia e alternativa
+  al lavoro in sede. Come tale, richiede la stessa attenzione e cura che mettiamo nel lavoro in sede.
+- Il lavoro da remoto richiede un impegno collettivo da parte di chi lavora da remoto e di chi lavora
+  in ufficio. Se uno di questi due gruppi non si impegna a sufficienza, non può funzionare.
+- Una filosofia remote-first ben realizzata migliora la qualità del lavoro di tutti, portando con sé
+  una maggiore calma e trasparenza nella comunicazione, nelle decisioni e nel lavoro di tutti i giorni.
+- Se stai lavorando da remoto, sei in ufficio. Se hai bisogno di fare altro, comunicalo al team e chiedi
+  un permesso. Lavorare mentre fai altro _non_ è accettabile, perché ne risente non solo la tua produttività,
+  ma anche quella delle persone con cui lavori.
+- Cerca di pianificare con più anticipo possibile il lavoro da remoto e ricorda ai tuoi colleghi che
+  lavorerai da remoto il giorno prima e il giorno stesso. Evita di non presentarti in ufficio e informare
+  tutti la mattina stessa (salvo casi eccezionali).
+- Adotta la trasparenza per default. Assicurati che tutte le comunicazioni avvengano su canali pubblici a
+  meno che non sia strettamente necessario tenerle private, in modo che tutti abbiano modo di tenersi
+  informati su quello che accade.
+
+### Spazio di lavoro
+
+Il nostro spazio di lavoro è il nostro tempio. Questo è vero tanto per il lavoro in sede quanto per 
+quello da remoto. La differenza  è che, mentre l'ufficio è uno spazio in cui tutti quanti ci occupiamo
+di creare un ambiente di lavoro confortevole, lavorando da remoto starà a te assicurarti di avere tutti
+gli strumenti e le condizioni per essere il più produttivo/a possibile.
+
+Ecco una lista delle cose indispensabili per lavorare da remoto:
+
+- **Una scrivania.** Questa non dev'essere una scrivania vera e propria, ma c'è comunque bisogno di
+  uno spazio dedicato esclusivamente al lavoro. Quando sei lì, sei in ufficio. Questo spazio dev'essere
+  dedicato, silenzioso, libero da distrazioni e altre persone.
+- **Una buona connessione a internet.** È frustrante essere in una riunione con qualcuno che non si 
+  riesce a capire o si disconnette in continuazione. Se hai intenzione di lavorare da remoto, 
+  investi in una buona connessione a internet. Se non sai come fare, troviamo una soluzione insieme.
+
+### Meeting da remoto (e non)
```

Comment:
Questa sezione la sposterei in una pagina a se stante in quanto come specificato anche nel titolo riguarda sia il lavoro da remoto che non.